sceptre/config/reader.py

```diff
@@ -475,7 +475,9 @@ def _collect_s3_details(stack_name, config):
         :rtype: dict
         """
         s3_details = None
-        if "template_bucket_name" in config:
+        # If the config explicitly sets the template_bucket_name to None, we don't want to enter
+        # this conditional block.
+        if config.get("template_bucket_name") is not None:
```

Comment:
I think we can shorten this to `if config.get("template_bucket_name"):`?

I believe config.get() defaults to `None` if it's not found and if template_bucket_name is explicity None then the condition wouldn't enter the block.

Reply:
You're right that None is falsey. However, I like to try to be as explicit in these checks as possible, since other values could be falsey too, like empty strings. I suppose in this case, we might want to treat an empty string the same as `None`. Nonetheless, when one is not explicit about what they're checking for, it can lead future maintainers to wonder about what sorts of values could be expected here.